iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -123,6 +123,7 @@ Inspector.prototype.updateURLFragment = function () {
 }
 
 Inspector.prototype.buildDisplayOptionsMenu = function () {
+    var inspector = this;
     this._optionsMenu.reset();
     this._optionsMenu.addCheckboxItem("Highlight", function (checked) { inspector.highlightAllTags(checked)}, "highlight-tags");
```

Comment:
This seems to partially revert commit 133c1092.  Use arrow function for consistency?
```
this._optionsMenu.addCheckboxItem("Highlight", (checked) => this.highlightAllTags(checked), "highlight-tags");
```